bundle/language_en.properties

```diff
@@ -0,0 +1,173 @@
+user.usergroupenotupdated = User group not updated.
+user.wrongusernameorpassword = Wrong username or password.
+user.unabletologuserlogininthesessionaudittable = Unable to log user login in the session_audit table.
+user.passwordtooshort = Pass word too short.
+user.usernamenotfound = Username not found. 
+opd.notefieldismandatory = Note field is mandatory.
+opd.isnotcreated = Opd is not created.
+opd.notfound = Opd not found.
+opd.isnotupdated = Opd is not updated.
+opd.isnotdeleted = Opd is not deleted.
+operation.isnotcreated = Operation is not created.
+operation.notfound = Operation not found.
+operation.isnotupdated = Operation is not updated.
+operation.atleastonefieldbetweenadmissionandopdisrequired = At least one field between admission and Opd is required.
+operation.operationrowisnotcreated = OperationRow is not created.
+operation.operationrownotfound = OperationRow not found.
+operation.operationrowisnotupdated = OperationRow is not updated.
+operation.operationrowisnotdeleted = OperationRow is not deleted.
+operationtype.notfound = Operation type not found.
+operationtype.isnotcreated = Operation type is not created.
+operationtype.isnotupdated = Operation type is not updated.
+patient.notfound = Patient not found.
+patient.fieldisrequired = Patient field is required.
+patient.codemismatch = Patient code mismatch.
+patient.isnotcreated = Patient is not created.
+patient.isnotupdated = Patient is not updated.
+patient.isnotdeleted = Patient is not deleted.
+patient.arenotmerged = PAtient are not merged.
+patient.doesnotexist = Patient does not exist.
+patient.notefieldismandatory = Note field is mandatory.
+patientvaccine.isnotcreated = Patient Vaccine is not created.
+patientvaccine.isnotupdated = Patient Vaccine is not updated.
+patientvaccine.isnotdeleted = Patient vaccine is not deleted.
+permission.notfound = Permission not found.
+permission.isnotupdated = Permission is not updated.
+pregnanttreattype.notfound = Pregnant treatment type not found.
+pregnanttreattype.isnotcreated = Pregnant treatment type is not created.
+pregnanttreattype.isnotupdated = Pregnant treatment type is not updated.
+pricelist.hasnotbeenduplicated = Price list has not been duplicated.
+pricelist.isnotupdated = Price list is not updated.
+pricelist.isnotcreated = Price list is not created.
+pricesothers.isnotupdated = Prices Others is not updated.
+pricesothers.isnotcreated = Prices Others is not created.
+sms.somesmsarenotfound = Some Sms are not found.
+stats.filenotfound = File not found.
+stats.failedtoloadthefiletype = Failed to load the files type.
+supplier.notfound = Supplier not found.
+supplier.isnotcreated = Supplier is not created.
+supplier.isnotupdated = Supplier is not updated.
+therapy.arenotreplaced = Therapies are not replaced.
+therapy.arenotdeleted = Therapies are not deleted.
+vaccine.isnotcreated = Vaccine is not created.
+vaccine.isnotupdated = Vaccine is not updated.
+vaccine.isnotdeleted = vaccine is not updated.
```

Comment:
There are so many things that need to be changed in this PR but I'll  only note one of them here.  The key is includes the text `deleted` but the property says `updated`